types/address.go

```diff
@@ -157,28 +153,6 @@ func AccAddressFromHexUnsafe(address string) (addr AccAddress, err error) {
 	return AccAddress(bz), err
 }
 
-// VerifyAddressFormat verifies that the provided bytes form a valid address
-// according to the default address rules or a custom address verifier set by
-// GetConfig().SetAddressVerifier().
-// TODO make an issue to get rid of global Config
-// ref: https://github.com/cosmos/cosmos-sdk/issues/9690
-func VerifyAddressFormat(bz []byte) error {
-	verifier := GetConfig().GetAddressVerifier()
-	if verifier != nil {
-		return verifier(bz)
-	}
```

Comment:
i think we need to keep this function and the verification in other address functions as not everyone will migrate right away. We would remove functionality to set custom verifiers but verify the basic we do now

Reply:
People can use an address codec to do so tbh. It is being populated everywhere in v0.51 (clientCtx, sign mode context, keepers).
Keeping it around could lead to a divergence of the address codec verification and the one in the global config.